devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -96,7 +96,7 @@ def parse(String description) {
 private void createChildDevices() {
 	def i = 2
 	addChildDevice("Child Switch Health", "${device.deviceNetworkId}:0${i}", device.hubId,
-			[completedSetup: true, label: "${device.displayName.split("1")[-1]}${i}", isComponent : false])
+			[completedSetup: true, label: "${device.displayName[0..-2]}${i}", isComponent : false])
```

Comment:
Might be good to add a space or a hyphen?

Reply:
`device.displayName` will end with a ` 1` so this will change it to ` 2`

Reply:
ah yes, -2 will be the space before 1

my bad